ansible/roles/cartodb.postgresql/tasks/postgis.yml

```diff
@@ -4,8 +4,10 @@
    apt: name={{ item }} state=present
    with_items:
     - libxml2-dev
-    - postgresql-9.3-postgis-2.2
-    - postgresql-9.3-postgis-scripts
+    - liblwgeom-2.1.8
```

Comment:
This was causing a fresh install to bomb. Do you know why this and `postgis` are needed in addition to the last two packages?